oak/server/rust/oak_runtime/src/runtime/introspect.rs

```diff
@@ -170,6 +172,24 @@ async fn make_server(
         let _ = termination_notificiation_receiver.await;
     });
 
+    // Listen for introspection events
+    let introspection_event_receiver = runtime.introspection_event_sender.clone().subscribe();
```

Comment:
Could you elaborate on why the clone call is necessary, in a comment? Also, could this be moved inside the async closure below?

Reply:
Actually the clone is not needed, moved it inside of the closure. :)